dask_geomodeling/geometry/aggregate.py

```diff
@@ -320,6 +320,7 @@ def get_sources_and_requests(self, **request):
             "bbox": (x1, y1, x2, y2),
             "width": width,
             "height": height,
+            "time_resolution": request.get("time_resolution"),
```

Comment:
This adds time_resolution to every request and it could be None. It would be less invasive to only add the item to the raster request if it is present in the original request.

Reply:
Never mind, I was fooled by the variable name 'req'